conpr/08_LE_Tasks/src/le/fj/Sum.java

```diff
@@ -20,7 +20,7 @@ public class Sum extends RecursiveTask<Integer> {
     protected Integer compute() {
         if (hi - lo < THRESHOLD) {
             int sum = array[lo];
-            for(int i = lo + 1; i <= lo; i++) {
+            for(int i = lo + 1; i < hi; i++) {
```

Comment:
This was a mistake in the original for loop that caused it never to run, and thus only the first elements of each sub-array in the recursion tree were actually being summed.